homeassistant/components/wiz/translations/nl.json

```diff
@@ -7,19 +7,29 @@
         "error": {
```

Comment:
These will disappear one rebase. Its just out of sync with `dev` because of the dep chain